core/block_svg.js

```diff
@@ -1191,11 +1191,10 @@ Blockly.BlockSvg.prototype.setHighlighted = function(highlighted) {
   if (highlighted) {
     this.svgPath_.setAttribute('filter',
         'url(#' + this.workspace.options.embossFilterId + ')');
-    this.svgPathLight_.style.display = 'none';
   } else {
-    this.svgPath_.removeAttribute('filter');
-    delete this.svgPathLight_.style.display;
+    this.svgPath_.setAttribute('filter', 'none');
   }
+  this.svgPathLight_.style.display = 'none';
```

Comment:
This line deletes the top-left highlight on every block when executed and never restores it -- as can be seen in your demo.

If 'delete' isn't working, then set the display to 'inline'.

Reply:
Thanks for the review 😄

> If 'delete' isn't working, then set the display to 'inline'.

In https://github.com/google/blockly/pull/2580/commits/09b8389d7c9e1ed47f042a5f141e24bb0b0ceac8, change the display style 'none' to 'inline'.